livre/eq/render/RayCastRenderer.cpp

```diff
@@ -203,6 +206,9 @@ struct RayCastRenderer::Impl
         tParamNameGL = glGetUniformLocation( program, "nSamplesPerRay" );
         glUniform1i( tParamNameGL, _computedSamplesPerRay );
 
+        tParamNameGL = glGetUniformLocation( program, "nDefaultSamplesPerRay" );
+        glUniform1i( tParamNameGL, nDefaultSamplesPerRay );
+
```

Comment:
Not an easy one. CMake magic is needed while shader is converted into an header or improve the glshader loader to do search and replace in the shader string. I am not sure for one variable any of those worth. If more and more needed surely it can be done.